addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -350,6 +351,18 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                 .attr('data-editor-message-default', true)
                 .attr('data-editor-message', this.env._t('DRAG BUILDING BLOCKS HERE'));
         $wrap.filter(':empty').attr('contenteditable', false);
+        for (let htmlEl of $wrap.not("[placeholder]").filter('[data-oe-sanitize="no_block"]')) {
```

Comment:
Note that `$wrap` is such a bad naming, but it was like that before your pr so nevermind.
But could have been changed, it really looks like it's holding the `<div id="wrap"/>` jQuery element.